runtime/lua/vim/ui/notification.lua

```diff
@@ -0,0 +1,105 @@
+local utils = require('vim.ui._utils')
+
+vim.api.nvim_command("hi NotificationInfo guifg=#80ff95")
+vim.api.nvim_command("hi NotificationWarning guifg=#fff454")
+vim.api.nvim_command("hi NotificationError guifg=#c44323")
+
```

Comment:
do you want me to comment on things like this that should be placed elsewhere or just wait?